python/pyspark/sql/tests/test_udf.py

```diff
@@ -804,6 +804,54 @@ def test_udf_with_rand(self):
         )
 
 
+class UDFTests(BaseUDFTests, ReusedSQLTestCase):
+    @classmethod
+    def setUpClass(cls):
+        super(BaseUDFTests, cls).setUpClass()
+        cls.spark.conf.set("spark.sql.execution.pythonUDF.arrow.enabled", "false")
+
+
+def test_use_arrow(self):
```

Comment:
@xinrong-meng Who runs this test?

Reply:
Good catch! Removed it https://github.com/apache/spark/pull/40725/commits/63ef94e6855a9b9cbf7f20282f28cebaf06cd3c6. `test_use_arrow` is duplicated in PythonUDFArrowTestsMixin of test_arrow_python_udf.py.